### DISCUSSION TITLE
The Real Heroes: Appreciation And Compliments Message

### DISCUSSION THREAD

**Discussion by ferreira-ff** [Whisky-App/Whisky#302] [General]
**Please; do not take this message, as any kind of spam. I have no affiliation or sponsorship from anyone, or any project on GitHub.**

_**To the real Heroes of macOS's gaming porting development:** the Wine developers, Mr. Marovitz, the creator of Whisky and Mr. Greer, the creator of Wineskin; of course, all other contributors deserve my compliments too.
Without all of you, gaming on Mac would slowly go to oblivion..._

Show your appreciation here; regarding donations, only if you feel you want to and can, but not here.

THANK YOU and kind regards to all

**Comment by odahcam** [comment]
Thank you for this amazing project! I'm sure looking forward to contribute.
